content/partials/specs/versions-macos-xcode-12-0.md

```diff
@@ -94,7 +94,7 @@ weight: 5
 - iOS 13.5
 - iOS 13.6
 - iOS 13.7
-- iOS 14.4
+- iOS 14.3

```

Comment:
are you sure about this change?

Reply:
yes, this is what we have installed in the base image, here is the full output of `xcode-project test-destinations`

```
List available test devices
Runtime: iOS 11.4
- iPad (5th generation)
- iPad (6th generation)
- iPad Air
- iPad Air 2
- iPad Pro (10.5-inch)
- iPad Pro (12.9-inch) (1st generation)
- iPad Pro (12.9-inch) (2nd generation)
- iPad Pro (9.7-inch)
- iPhone 5s
- iPhone 6
- iPhone 6 Plus
- iPhone 6s
- iPhone 6s Plus
- iPhone 7
- iPhone 7 Plus
- iPhone 8
- iPhone 8 Plus
- iPhone SE (1st generation)
- iPhone X
Runtime: iOS 12.4
- iPad (5th generation)
- iPad (6th generation)
- iPad Air
- iPad Air (3rd generation)
- iPad Air 2
- iPad Pro (10.5-inch)
- iPad Pro (11-inch) (1st generation)
- iPad Pro (12.9-inch) (1st generation)
- iPad Pro (12.9-inch) (2nd generation)
- iPad Pro (12.9-inch) (3rd generation)
- iPad Pro (9.7-inch)
- iPhone 5s
- iPhone 6
- iPhone 6 Plus
- iPhone 6s
- iPhone 6s Plus
- iPhone 7
- iPhone 7 Plus
- iPhone 8
- iPhone 8 Plus
- iPhone SE (1st generation)
- iPhone X
- iPhone Xs
- iPhone Xs Max
- iPhone Xʀ
Runtime: iOS 13.0
- iPad Air (3rd generation)
- iPad Pro (11-inch) (1st generation)
- iPad Pro (12.9-inch) (3rd generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 8
- iPhone 8 Plus
Runtime: iOS 13.1
- iPad Air (3rd generation)
- iPad Pro (11-inch) (1st generation)
- iPad Pro (12.9-inch) (3rd generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 8
- iPhone 8 Plus
Runtime: iOS 13.2
- iPad (7th generation)
- iPad Air (3rd generation)
- iPad Pro (11-inch) (1st generation)
- iPad Pro (12.9-inch) (3rd generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 8
- iPhone 8 Plus
Runtime: iOS 13.3
- iPad (7th generation)
- iPad Air (3rd generation)
- iPad Pro (11-inch) (1st generation)
- iPad Pro (12.9-inch) (3rd generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 8
- iPhone 8 Plus
Runtime: iOS 13.4
- iPad (7th generation)
- iPad Air (3rd generation)
- iPad Pro (11-inch) (2nd generation)
- iPad Pro (12.9-inch) (4th generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 8
- iPhone 8 Plus
Runtime: iOS 13.5
- iPad (7th generation)
- iPad Air (3rd generation)
- iPad Pro (11-inch) (2nd generation)
- iPad Pro (12.9-inch) (4th generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 8
- iPhone 8 Plus
- iPhone SE (2nd generation)
Runtime: iOS 13.6
- iPad (7th generation)
- iPad Air (3rd generation)
- iPad Pro (11-inch) (2nd generation)
- iPad Pro (12.9-inch) (4th generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 8
- iPhone 8 Plus
- iPhone SE (2nd generation)
Runtime: iOS 13.7
- iPad (7th generation)
- iPad Air (3rd generation)
- iPad Pro (11-inch) (2nd generation)
- iPad Pro (12.9-inch) (4th generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 8
- iPhone 8 Plus
- iPhone SE (2nd generation)
Runtime: iOS 14.3
- iPad (8th generation)
- iPad Air (4th generation)
- iPad Pro (11-inch) (2nd generation)
- iPad Pro (12.9-inch) (4th generation)
- iPad Pro (9.7-inch)
- iPhone 11
- iPhone 11 Pro
- iPhone 11 Pro Max
- iPhone 12
- iPhone 12 Pro
- iPhone 12 Pro Max
- iPhone 12 mini
- iPhone 8
- iPhone 8 Plus
- iPhone SE (2nd generation)
- iPod touch (7th generation)
Runtime: tvOS 11.3
- Apple TV
- Apple TV 4K
- Apple TV 4K (at 1080p)
Runtime: tvOS 11.4
- Apple TV
- Apple TV 4K
- Apple TV 4K (at 1080p)
Runtime: tvOS 12.0
- Apple TV
- Apple TV 4K
- Apple TV 4K (at 1080p)
Runtime: tvOS 13.0
- Apple TV
- Apple TV 4K
- Apple TV 4K (at 1080p)
Runtime: tvOS 13.2
- Apple TV
- Apple TV 4K
- Apple TV 4K (at 1080p)
Runtime: tvOS 13.3
- Apple TV
- Apple TV 4K
- Apple TV 4K (at 1080p)
Runtime: tvOS 13.4
- Apple TV
- Apple TV 4K
- Apple TV 4K (at 1080p)
Runtime: tvOS 14.3
- Apple TV
- Apple TV 4K
- Apple TV 4K (at 1080p)
Runtime: watchOS 6.2
- Apple Watch Series 4 - 40mm
- Apple Watch Series 4 - 44mm
- Apple Watch Series 5 - 40mm
- Apple Watch Series 5 - 44mm
Runtime: watchOS 7.0
- Apple Watch Series 5 - 40mm
- Apple Watch Series 5 - 44mm
- Apple Watch Series 6 - 40mm
- Apple Watch Series 6 - 44mm
Runtime: watchOS 7.2
- Apple Watch Series 5 - 40mm
- Apple Watch Series 5 - 44mm
- Apple Watch Series 6 - 40mm
- Apple Watch Series 6 - 44mm
```